torch/autograd/gradcheck.py

```diff
@@ -267,7 +265,8 @@ def new_func(*input_args):
         input_args = input_args[:-len(grad_outputs)]
         outputs = _differentiable_outputs(func(*input_args))
         input_args = tuple(x for x in input_args if isinstance(x, torch.Tensor) and x.requires_grad)
-        grad_inputs = torch.autograd.grad(outputs, input_args, grad_outputs, create_graph=True)
+        grad_inputs = torch.autograd.grad(outputs, input_args, grad_outputs,
+                                          create_graph=True, only_inputs=True, allow_unused=True)
```

Comment:
This got an extra `allow_unused` flag - why? Also, `only_inputs` is `True` by default IIRC, so no need to specify it.